### ISSUE TITLE
oUF / oUF_Neav

### ISSUE THREAD

**Issue by frankallen** [L3n1n/NeavUI-Wrath#1]
Any plans of adding them soon? (and the SquidMod please!)

**Comment by L3n1n** [comment]
No, i have no plans because like i said early i prefer to play without oUF. But i know there is some people who love to play with oUF. And I will do it.

Big problem is lack of a normal working version oUF for Wrath. I hard codded (oUF and oUF_Neav at the same time) and now it works but i didn't test all functionality because i am solo player. Some pieces of code take a very long time to figure out and I have to debug them all the time. I'm not senior programmer and even middle, but i code with love.

**Comment by frankallen** [comment]
I could test it in the upcoming week/weekend for group/raid playing.
